### ISSUE TITLE
Accessibility: Color Contrast Failure on Five for the Future Program Expectations Page

### ISSUE THREAD

**Issue by amberhinds** [WordPress/wordpress.org#267]
Can someone fix this blue on blue link on https://wordpress.org/five-for-the-future/expectations/

<img width="675" alt="Screenshot 2024-04-16 at 9 16 28 PM" src="https://github.com/WordPress/wordpress.org/assets/13153456/6d422363-54f1-43a3-b39f-25d3248949da">

@Otto42 is that something you have access to?

**Comment by ryelle** [comment]
I've just fixed that so it's readable, though the hover color contrast is lower. It wasn't passing AA before though, so I think we can let it be until we get to this section for the redesign/refresh.
